overlord/ifacestate/ifacemgr.go

```diff
@@ -56,6 +57,10 @@ func Manager(s *state.State, hookManager *hookstate.HookManager, extraInterfaces
 		return nil, err
 	}
 
+	s.Lock()
```

Comment:
If this is the only call site the callee could have locked.

Reply:
it's part of a family of functions (Replace*) that all don't lock, for symmetry with other Set/Get functions we have